pkg/controller/plan/migration.go

```diff
@@ -1638,6 +1647,19 @@ func (r *Migration) updatePopulatorCopyProgress(vm *plan.VMStatus, step *plan.St
 	return
 }
 
+func (r *Migration) isPopulatorFailed(populatorPods []core.Pod, givenPvcId string) error {
+	for _, pod := range populatorPods {
+		pvcId := strings.Split(pod.Name, "populate-")[1]
```

Comment:
how about extracting `popoulate-` to a constant and use it also in getPopulatorPods?
also `Split` is a bit cumbersome for this purpose, why not to slice the string?